app-modules/interaction/database/factories/InteractionFactory.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace Assist\Interaction\Database\Factories;
+
+use App\Models\User;
+use Assist\Prospect\Models\Prospect;
+use Assist\AssistDataModel\Models\Student;
+use Assist\Interaction\Models\InteractionType;
+use Assist\Interaction\Models\InteractionDriver;
+use Assist\Interaction\Models\InteractionStatus;
+use Assist\Interaction\Models\InteractionOutcome;
+use Assist\Interaction\Models\InteractionCampaign;
+use Assist\Interaction\Models\InteractionRelation;
+use Assist\ServiceManagement\Models\ServiceRequest;
+use Illuminate\Database\Eloquent\Factories\Factory;
+use Assist\Interaction\Models\InteractionInstitution;
+
+/**
+ * @extends \Illuminate\Database\Eloquent\Factories\Factory<\Assist\Interaction\Models\Interaction>
+ */
+class InteractionFactory extends Factory
+{
+    public function definition(): array
+    {
+        $interactable = fake()->randomElement([
+            Student::class,
+            Prospect::class,
+            ServiceRequest::class,
```

Comment:
Interesting, why would a `ServiceRequest` be an interactable?

Reply:
I'm not sure - this was part of the ticket requirement.

Reply:
Oh shoot, I apologize, I didn't see that at the very end of the ticket. Np